extensions/amp-story-subscriptions/0.1/amp-story-subscriptions.js

```diff
@@ -124,20 +149,64 @@ export class AmpStorySubscriptions extends AMP.BaseElement {
     if (showDialog) {
       // This call would first retrieve entitlements that are already fetched from publisher backend when page loads.
       // If the response is granted, do nothing. If the response is not granted, the paywall would be triggered.
-      return this.subscriptionService_.maybeRenderDialogForSelectedPlatform();
+      return this.subscriptionService_
+        .maybeRenderDialogForSelectedPlatform()
+        .then(() => {
+          if (this.viewer_.isEmbedded()) {
+            setTimeout(() => {
+              const buttonEl = this.win.document.querySelector(
+                'amp-subscriptions-dialog .i-amphtml-story-subscriptions-dialog-banner-button'
+              );
+              buttonEl &&
+                this.mutateElement(() =>
+                  buttonEl.classList.add(
+                    'i-amphtml-story-subscriptions-dialog-banner-button-visible'
+                  )
+                );
+            }, SKIP_BUTTON_DELAY_DURATION);
+          }
+        });
     }
     this.subscriptionService_.getDialog().close();
   }
 
+  /**
+   * @param {!Event} event
+   * @private
+   */
+  onSkipButtonClick_(event) {
+    if (
+      event.target.classList.contains(
+        'i-amphtml-story-subscriptions-dialog-banner-button-visible'
+      )
+    ) {
+      const advancementMode = this.storeService_.get(
+        StateProperty.ADVANCEMENT_MODE
+      );
+      this.viewerMessagingHandler_.send('selectDocument', {
+        'next': true,
+        'advancementMode': advancementMode,
+      });
+    }
+  }
+
   /**
    * @return {!Element}
    * @private
-   * TODO(#37285): add "next story" button to the banner
    */
   renderSubscriptionsDialogTemplate_() {
     return (
       <div subscriptions-dialog subscriptions-display="NOT granted">
-        <div class="i-amphtml-story-subscriptions-dialog-banner"></div>
+        <div class="i-amphtml-story-subscriptions-dialog-banner">
+          <span
+            class="i-amphtml-story-subscriptions-dialog-banner-button"
+            role="button"
```

Comment:
This element is not tabable. 
This needs to have [`tabindex="0"`](https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/Roles/button_role) or be a button element.
button elements give us all of the a11y benefits by default such as `tab-index` and `role` by default so I would use that. If using that you will need to remove default button styles from the element.

I just realized the other elements have this problem. Filed #38110 to fix them in a follow up.

Reply:
Done. Cool thanks for filing the bug!